pkg/metrics/collector/pod_ip_metrics.go

```diff
@@ -23,18 +23,19 @@ import (
 	"os"
 	"strings"
 
+	"github.com/fsnotify/fsnotify"
 	"github.com/golang/glog"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 type ipFamily int
 
 const (
-	gkePodNetworkDir = "/host/var/lib/cni/networks/gke-pod-network"
-	ipv4 ipFamily = iota
-	ipv6 = 1
-	dual = 2
-	dualStack = "IPV4_IPV6"
+	gkePodNetworkDir          = "/host/var/lib/cni/networks/gke-pod-network"
```

Comment:
May as well update to the following to take advantage of iota.

const (
	ipv4 ipFamily = iota
	ipv6
	dual

	gkePodNetworkDir = "/host/var/lib/cni/networks/gke-pod-network"
	dualStack                = "IPV4_IPV6"
)